build/validate_links.py

```diff
@@ -16,6 +21,29 @@ def parse_links(filename):
     links = [raw_link[0] for raw_link in raw_links]
     return links
 
+def dup_links(links):
+    """Check for duplicated links"""
+    print(f'Checking for duplicated links...')
+    hasError = False
+    seen = {}
+    dupes = []
+
+    for x in links:
+        if x in ignored_links:
+            continue
+
+        if x not in seen:
+            seen[x] = 1
+        else:
+            if seen[x] == 1:
+                dupes.append(x)
```

Comment:
Many URLs contain the `/` character at the end. This section does not consider them.

This is returning at the moment:
``` bash
Found duplicated links: ['https://github.com/davemachado/public-api']
```

If you consider the character `/` at the end:
``` bash
Found duplicated links: ['https://mailboxlayer.com', 'https://github.com/davemachado/public-api', 'https://languagelayer.com']
```

The `str.rstrip()` function solves this.

```suggestion
        x = x.rstrip('/')
        if x in ignored_links:
            continue

        if x not in seen:
            seen[x] = 1
        else:
            if seen[x] == 1:
                dupes.append(x)
```

Reply:
@matheusfelipeog Thanks for reviewing my PR and spotting an extra check with `/` character.